### DISCUSSION TITLE
I want to adjust the FFMPG settings for M4B output to specific audio parameters, please include this

### DISCUSSION THREAD

**Discussion by Allmight97** [openaudible/openaudible#1192] [General]
Hi - your software is just about perfect what I need to do and I have supported it through the last couple of major versions.

I'd like some setttings (or tell me how to do it manually) to adjust the audio parameters of m4b files.  e.g. I want to be about to set the sample rates, the channels (mono vs stereo) and the bitrates.  I like to keep my audiobook library size in check with the following parameters: 32 or 22050k sample rate, 64bit, mono.  Your software does everything I need in one package except for that.

**Comment by openaudible** [comment]
There is an undocumented feature for this. 
If you create a file called ffmpeg_args_mp3.txt or ffmpeg_args_m4b.txt and save it in your main settings directory (next to the settings.json file) each line of the file will be added as an extra argument.

For instance, to force 44100 sample rate and mono, you'd put:

```
-ar
44100
-ac
1
```
Then take a look at the log file and make sure the ffmpeg command that is being used is what you expect.  You'll need to adjust the file to make all the changes you're looking to make.
